commons/src/main/java/com/flipkart/flux/commons/dto/Eventsinfo.java

```diff
@@ -0,0 +1,31 @@
+package com.flipkart.flux.commons.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@AllArgsConstructor
+public class Eventsinfo {
```

Comment:
1. Add the licensing details to all new files
2. Add relevant doc
3. Refactor camel casing EventsInfo